packages/core/src/login/webview/vue/login.vue

```diff
@@ -94,10 +94,12 @@
                             :itemTitle="existingLogin.title"
                             class="selectable-item"
                         ></SelectableItem>
+                        <div style="margin-bottom: 12px"></div>
```

Comment:
I don't think you need a new element, just apply the class I mentioned in my above comment to the element itself. `SelectableItem` in this case:

```
class="selectable-item bottomMargin"
```

Reply:
done